proceeding/tables/catalog_table.tex

```diff
@@ -0,0 +1,21 @@
+% Table
+\begin{table}[h]
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || lrll ||}
```

Comment:
I think you should also remove the outer vertical lines.
I've never seen this in publications, and it looks weird / unnecesary to me.